config/environments/preprod.rb

```diff
@@ -59,7 +59,7 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
-  # config.action_mailer.default_url_options[:host] = ENV["SERVER_FQDN"]
+  config.action_mailer.default_url_options[:host] = ENV["SERVER_FQDN"]
```

Comment:
Does this change want to be applied to staging and production as well?

Reply:
wow nice catch, recommenting